lib/blocks.php

```diff
@@ -131,6 +131,38 @@ function gutenberg_serialize_block( $block ) {
 	return $content;
 }
 
+/**
+ * Given an array of parsed blocks, returns the css styles they require.
+ *
+ * @since 1.8.1
+ *
+ * @param array $blocks Parsed block array.
+ *
+ * @return string CSS style block or empty string
+ */
+function compute_styles( $blocks ) {
+	$map_attrs_styles = array(
+		'textColor'       => 'color',
+		'backgroundColor' => 'background-color',
+	);
+	$styles           = '';
+	foreach ( $blocks as $block ) {
+		$attributes = is_array( $block['attrs'] ) ? $block['attrs'] : array();
+		if ( isset( $attributes['anchor'] ) ) {
+			$rules = '';
+			foreach ( $map_attrs_styles as $attr_key => $css_property ) {
+				if ( isset( $attributes[ $attr_key ] ) ) {
+					$rules .= "\n\t" . esc_attr( sprintf( '%1$s: %2$s;', $css_property, $attributes[ $attr_key ] ) );
```

Comment:
Do we need to do the whitespace prettifying here?

Reply:
Without this whitespace prettifying when we check the source code the styles are messy and all on the same line but they work. I'm not certain what we normally do in this cases.